examples/dataflow-streaming-schema-handler/src/main/java/com/google/cloud/pso/transforms/ReadPubSub.java

```diff
@@ -0,0 +1,34 @@
+package com.google.cloud.pso.transforms;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Iterator;
+import java.util.Map.Entry;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.values.KV;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ReadPubSub extends DoFn<String, KV<String, String>> {
+  private static final Logger LOG = LoggerFactory.getLogger(ReadPubSub.class);
+
+  @ProcessElement
+  public void processElement(ProcessContext context) {
+    String input = context.element();
+
+    JsonFactory factory = new JsonFactory();
+    ObjectMapper mapper = new ObjectMapper(factory);
```

Comment:
These objects could be initialized in the `startBundle` or `setup` methods. Initializing them in the `processElement` method is inefficient.

Reply:
Good catch, thanks!